manifests/install.pp

```diff
@@ -104,8 +104,8 @@
               'beta'   => 'https://packages.grafana.com/oss/rpm-beta',
             }
 
-            yumrepo { 'grafana':
-              descr    => 'grafana repo',
```

Comment:
Do we need to remove the old repo?

```puppet
yumrepo { 'grafana':
  ensure => absent,
}
```
??

Reply:
You are probably right. I will fix this in the evening.